doc/btp_gap.txt

```diff
@@ -645,6 +645,189 @@ Commands and responses:
 
                 In case of an error, the error response will be returned.
 
+	Opcode 0x2a - Configure Advertising Set
+
+		Controller Index:	<controller id>
+		Command parameters:	Advertising Set ID (1 octet)
+					SID (1 octet)
+					Own Address Type (1 octet)
+					Advertising Type (1 octet)
+					Primary Advertising PHY (1 octet)
+					Secondary Advertising PHY (1 octet)
+					Secondary Advertising Maximum Skip (1 octet)
+					Coded PHY Options (1 octet)
+					Filter Accept List (1 octet)
+					Directed Advertising (1 octet)
+					Options (2 octets)
+					Minimum Interval, in 0.625ms units (4 octets)
+					Maximum Interval, in 0.625ms units (4 octets)
+					Peer Address Type (1 octet)
+					Peer Address (6 octets)
+		Return Parameters:	Configured Advertising Set ID (1 octet)
+
+		This command is used to create and configure advertising sets, which can
+		be used for both legacy and extended advertisements.
+
+		If the Advertising Set ID is 0xFF, a new advertising set will be created.
```

Comment:
Why not treat it similar to HCI? 
i.e. if the SID is not already create, it will create a set with that ID, else it will (attempt) to reconfigure an existing advertising set